### PR TITLE
Integration of open paygo time based fonctionality

### PR DESCRIPTION
**Pull request by Peguy-WANDA**

### CODE REVIEW

File: firmware/src/credit.hpp [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #include "relay.hpp"
 
 float billing = 0.0;
-unsigned int tariff = 70;
+unsigned int tariff = 1;
```
Comment (dmohns) [inline firmware/src/credit.hpp:14]: Could you explain why this is needed?

File: firmware/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.pio
+.vscode/.browse.c_cpp.db*
+.vscode/c_cpp_properties.json
+.vscode/launch.json
+.vscode/ipch
```
Comment (dmohns) [inline firmware/.gitignore:5]: Please don't commit this `firmware/.gitignore` file. You can just delete the file locally, all exclusions you are adding are already tracked in the repository's root `.gitignore`.

**Review by dmohns — COMMENTED** [review]
2 small comments, otherwise LGTM!

File: firmware/.gitignore (hunk 2 above)
Comment (Peguy-WANDA) [reply firmware/.gitignore:5]: Ok daniel

**Pull request merged**